app/src/main/res/layout/view_teleconsult_doctor_info.xml

```diff
@@ -49,6 +49,28 @@
 
     </com.google.android.material.textfield.TextInputLayout>
 
+    <FrameLayout
+      android:id="@+id/signatureContainer"
+      android:layout_width="match_parent"
+      android:layout_height="@dimen/spacing_48"
+      android:layout_marginTop="@dimen/spacing_16"
+      android:background="@drawable/background_signature"
```

Comment:
`addSignatureButton` is a outline button, this frame is supposed to emulate that. Even if we set the background manually it's not compulsory for both of these to have same background 🤔 .

Reply:
I am unsure about that, If we want to do that we can no longer use `MaterialButton`, because it doesn't draw background drawables. I can set a `Foreground` to emulate the stroke but it's only for API 23.

Also I am not sure how many times this get's changed TBH. Since these buttons are emulating the outline text fields used in the rest of the screen. Let's consider this later on?

WDYT?

Reply:
Makes sense.